cargo-nextest/src/dispatch.rs

```diff
@@ -594,6 +594,7 @@ impl CargoOptions {
 /// Test runner options.
 #[derive(Debug, Default, Args)]
 #[clap(next_help_heading = "RUNNER OPTIONS")]
+#[clap(allow_negative_numbers = true)]
```

Comment:
Could you move this to just the specific argument where negative numbers are allowed? i.e. the `test_threads` argument.